crates/biome_grit_formatter/tests/specs/grit/patterns/list_pattern.grit.snap

```diff
@@ -0,0 +1,56 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: grit/patterns/list_pattern.grit
+---
+# Input
+
+```grit
+`var $x = [$numbers]`=>`var firstPrimes = [$numbers]`where{$numbers<:[`2`,`3`,`5`]}
+
+
+`var $x = [$numbers]`=>`var firstPrimes = [$numbers]`where{$numbers<:[`2`,`3`,`5`,`6`,`7`,`8`]}
```

Comment:
I suggest adding also **long** lists that go over the default line width, so you can see how it behaves.

For example, here you used `.fill`, which doesn't - if memory serves me right - breakdown the lists on multiple lines.

Reply:
Yes, that's correct. I'll update the tests to go over the default line length, and refactor the code so that it breaks down the list on multiple lines once it reaches a certain length. 

Thank you!